source/FreeRTOS_IP_Multicast.c

```diff
@@ -0,0 +1,983 @@
+/*
```

Comment:
Need to add this file to the target sources for `freertos_plus_tcp` in `source/CMakeLists.txt`

Reply:
Done